core/src/main/scala/cats/mtl/ApplicativeHandle.scala

```diff
@@ -20,13 +20,155 @@ trait ApplicativeHandle[F[_], E] extends FunctorRaise[F, E] with Serializable {
 
 private[mtl] trait ApplicativeHandleInstances {
 
-  implicit def applicativeHandleForMonadError[F[_], E](implicit F: MonadError[F, E]): ApplicativeHandle[F, E] =
-    new ApplicativeHandle[F, E] {
-      def applicative = F
-      def functor = F
+  implicit final def handleEitherT[M[_], E](implicit M: Monad[M]): ApplicativeHandle[EitherTC[M, E]#l, E] = {
+    new ApplicativeHandle[EitherTC[M, E]#l, E] {
```

Comment:
Now that we have kind projector, I think it's probably worth nuking all of these partially-applied aliases. Easy search/replace post merge.

Reply:
100%!